click/README.md

```diff
@@ -0,0 +1,269 @@
+# Click on Unikraft
+
+This application starts a [Click](https://github.com/sysml/clickos) program with Unikraft.
+Follow the instructions below to set up, configure, build and run Click.
+
+## Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.
+Note that you still need to install the [requirements](#requirements).
+
+For building and running everything for Qemu in `x86_64`, follow the steps below:
+
+```console
+./setup.sh
+UK_DEFCONFIG=$(pwd)/.config.click-qemu-x86_64 make defconfig
+make -j $(nproc)
+sudo ip link set dev virbr0 down 2> /dev/null
+sudo ip link del dev virbr0 2> /dev/null
+sudo ip link add dev virbr0 type bridge
+sudo ip address add 172.44.0.1/24 dev virbr0
+sudo ip link set dev virbr0 up
+
+sudo qemu-system-x86_64 \
+    -netdev bridge,id=en0,br=virbr0 -device virtio-net-pci,netdev=en0 \
+    -append "netdev.ipv4_addr=172.44.0.2 netdev.ipv4_gw_addr=172.44.0.1 netdev.ipv4_subnet_mask=255.255.255.0 --" \
+    -kernel workdir/build/click_qemu-x86_64 \
+    -initrd helloworld.click \
+    -nographic
+```
+
+This will configure, build and run the `click` application.
+
+The same can be done for Qemu in `AArch64`, by running the commands below:
+
+```console
+./setup.sh
+make properclean
+UK_DEFCONFIG=$(pwd)/.config.click-qemu-aarch64 make defconfig
```

Comment:
This should be replaced with a `wget` from the scripts branch, like in the other applications